### ISSUE TITLE
setVisualizationCullingBox not working

### ISSUE THREAD

**Issue by bsabiston** [NVIDIAGameWorks/PhysX#49]
How does setVisualizationCullingBox work in PhysX? I tried setting a box to cull my debug lines, but it seems to have no effect.

**Comment by PierreTerdiman** [comment]
IIRC you also need to set PxVisualizationParameter::eCULL_BOX. The API could be improved here........

**Comment by bsabiston** [comment]
Yes, I looked through the code and saw that.  It should be in the documentation, man.  Thanks very much for the reply!

**Comment by amoravanszky** [comment]
I would actually prefer leaving the documentation as-is and removing the need to raise eCULL_BOX.  That should only control if this box is itself visualized.  I filed this in our internal bug tracker.

**Comment by PierreTerdiman** [comment]
I think my comment was actually wrong. We fixed the design issue with eCULL_BOX some time ago already. So these days eCULL_BOX should only control the debug vizualisation of the culling box itself, but you shouldn't need to enable that flag to make the culling work. So your call to setVisualizationCullingBox alone should be enough. If it doesn't work, then there is another issue somewhere.
